benchmarks/torchbench/benchtest.yaml

```diff
@@ -33,14 +33,356 @@ defaults: &defaults
     -t: train
```

Comment:
Attempted to add the flops metric by add the `--flops: True` flag to the `argv` list. It appears however that this feature is broken until something gets added to the new https://github.com/pytorch/benchmark/blob/main/torchbenchmark/util/extra_args.py#L79-L83 as the flag fails to pass there for all models.

Reply:
We could always add it on a per-test basis. How would it transmit the information to milabench, though?

Reply:
It can be accessed at the end of the model run through `model.get_flops()` (when available):
https://github.com/pytorch/benchmark/blob/c60f7eddaeaa348867e275343b145f441d791730/torchbenchmark/util/framework/vision/model_factory.py#L34-L44

I'll start looking for an implementation

Reply:
If `model` is given, an instrument could probably get the flops at the end, something like this:

```python
yield ov.phases.load_script
models = probe("...").first().accum()
yield ov.phases.run_script
model = models[0]
ov.give(flops=model.get_flops())
```

Reply:
I made this attempt but it fails with `TypeError: <lambda>() missing 1 required positional argument: 'cpu_walltime'`

```python
    (
        ov.probe("//run_one_step(func, cpu_walltime) > #exit")
        .kmap(model=lambda func: func.__self__)
        .kmap(model_flops=lambda model: model.get_flops() if hasattr(model, "flops") and hasattr(model, "get_flops") else (0, 0))
        .kmap(tflops=lambda model_flops, cpu_walltime: model_flops[0] * model_flops[1] / (cpu_walltime / 1.0e9) / 1.0e12)
        .give()
    )
```

Reply:
I think you want to use `augment`. `kmap` returns a fresh dictionary with the given keys, so `cpu_walltime` is thrown out. `augment` adds new keys.

Reply:
Got it working with `argument` thanks! Not commiting this as torchbench definitely broke their flops feature in the branch we use and models for which flops did work at some point (`resnet50`) now crash when attempting to enable `--flops`.

```python
    (
        ov.probe("//run_one_step(func, cpu_walltime) > #exit")
        .augment(model=lambda func: func.__self__)
        .augment(flops=lambda model: model.get_flops()[0] if hasattr(model, "flops") and hasattr(model, "get_flops") else 0)
        .augment(batch_size=lambda model: model.get_flops()[1] if hasattr(model, "flops") and hasattr(model, "get_flops") else 0)
        .kmap(tflops=lambda flops, batch_size, cpu_walltime: flops * batch_size / (cpu_walltime / 1.0e9) / 1.0e12)
        .give()
    )
```